### PR TITLE
Preserve cloud config

### PR DESCRIPTION
**Pull request by mrampton**

### CODE REVIEW

File: src/install.sh [hunk 1]
```diff
@@ -609,25 +603,18 @@ function create_k8s_custom_config() {
...
         gsutil cp "${config_file}" "${config_gs_path}"
       else
         echo "Using existing custom-credentials.json..."
         gsutil cp "${config_gs_path}" "${config_file}"
-
       fi
-
-
-
-      gsutil cp "${config_file}" "gs://${ARMORY_CONF_STORE_BUCKET}/front50/config_v2/config.json"
```
Comment (kevinawoo) [inline src/install.sh]: I think this was removed by accident.

File: src/install.sh [hunk 2]
```diff
@@ -576,30 +574,26 @@ function create_k8s_custom_config() {
...
       fi
-      if [[ "${OVERWRITE_CONFIG}" == "y" ]] ; then
+      if [[ "${OVERWRITE_CONFIG}" != "n" ]] ; then
         echo "Overwriting config.json..."
         aws --profile "${AWS_PROFILE}" --region us-east-1 s3 cp \
           "${config_file}" \
           "${config_s3_path}"
       else
-        echo "Using existing custom-credentials.json..."
+        echo "Using existing config.json..."
```
Comment (kevinawoo) [inline src/install.sh]: i.... think i'm confused with what this function is doing, and what `echo` is doing...

File: src/install.sh [hunk 3]
```diff
@@ -576,30 +574,26 @@ function create_k8s_custom_config() {
...
     local config_gs_path="gs://${ARMORY_CONF_STORE_BUCKET}/front50/config_v2/config.json"
     if [[ "${CONFIG_STORE}" == "S3" ]]; then
-      aws --profile "${AWS_PROFILE}" --region us-east-1 s3 ls "${config_s3_path}" > /dev/null 2>&1
-      result=$?
-      if [[ $result -eq 0 ]]; then
+      if aws --profile "${AWS_PROFILE}" --region us-east-1 s3 ls "${config_s3_path}" > /dev/null 2>&1; then
         get_var "config.json already exists, would you like to overwrite it? [y/n]: " OVERWRITE_CONFIG
       fi
-      if [[ "${OVERWRITE_CONFIG}" == "y" ]] ; then
+      if [[ "${OVERWRITE_CONFIG}" != "n" ]] ; then
```
Comment (kevinawoo) [inline src/install.sh]: i think this condition should be nested in the one above

Comment (kevinawoo) [inline src/install.sh]: let's set `OVERWRITE_CONFIG` to be defaultly `y` and if it exist, ask the user.

File: src/install.sh (hunk 1 above)
Comment (mrampton) [reply src/install.sh]: Looks like removing it is correct.

**Pull request merged**